src/components/QuestionSubmissions.tsx

```diff
@@ -0,0 +1,98 @@
+import Head from "next/head";
+import router from "next/router";
+import { useSession } from "next-auth/react";
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+dayjs.extend(relativeTime);
+
+// TODO: modify to return submissions of a specific question
+// maybe of a specifc user (depending on where you want user to be able
+// to view all submission to a questions or just their submission)
+const QuestionSubmissions = ({ userId }: { userId: string }) => {
```

Comment:
Can use with auth wrapper for these?

Reply:
hmm yea prolly should as discussed ill read up on that then replace everythign in one go yea